joomla_v4/translations/core/installation/language/de-LU/langmetadata.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <metafile client="installation" version="4.0">
 	<name>German (Luxembourg)</name>
-	<version>4.0.0</version>
-	<creationDate>11.08.2021</creationDate>
+	<version>4.0.3v1</version>
+	<creationDate>14.09.2021</creationDate>
```

Comment:
Should be September 2021
See: https://github.com/joomla/core-translations/blob/main/joomla_v4/source/installation/language/en-GB/langmetadata.xml#L5

Reply:
```suggestion
	<creationDate>September 2021</creationDate>
```